modules/apps/layout/layout-reports-web/src/main/java/com/liferay/layout/reports/web/internal/portlet/action/GetLayoutReportsIssuesMVCResourceCommand.java

```diff
@@ -167,13 +166,13 @@ private JSONObject _fetchLayoutReportIssuesJSONObject(
 		return JSONUtil.put(
 			"issues",
 			JSONUtil.putAll(
```

Comment:
@JanisZhang `JSONUtil.putAll()` creates a JSON array, so it's duplicate with the `JSONUtil.toJSONArray()` here.